ctl/sim_ctl/sim_config.sh

```diff
@@ -16,13 +16,12 @@ echo "###"
 # General
 ####################
 
-# create and clean-up sim_dir
+# create a new simulation run directory
 echo "simdir: "$sim_dir
-if [ -d "${sim_dir}" ]; then
+if [ -e "${sim_dir}" ]; then
```

Comment:
maybe I miss something, but why did you change `-d` to `-e`? It should be a directory.

Reply:
Any type of file in the way will thus be renamed.

Reply:
So concretely, the script would continue without problems, no matter what `$sim_dir` is.

Downside of checking only if it is a directory, the script would fail when it is a regular file.  Worse yet, it would continue with an old (supposedly) directory when it is a symlink.

It is a stronger check and moving anything out of the way (without destroying traces of any potential intentions by the user) is good.